core/schema/container.go

```diff
@@ -441,11 +441,33 @@ func (s *containerSchema) Install() {
 
 		dagql.Func("withDefaultTerminalCmd", s.withDefaultTerminalCmd).
 			Doc(`Set the default command to invoke for the container's terminal API.`).
-			ArgDoc("args", `The args of the command.`),
+			ArgDoc("args", `The args of the command.`).
+			ArgDoc("experimentalPrivilegedNesting",
+				`Provides dagger access to the executed command.`,
+				`Do not use this option unless you trust the command being executed;
+			the command being executed WILL BE GRANTED FULL ACCESS TO YOUR HOST
+			FILESYSTEM.`).
+			ArgDoc("insecureRootCapabilities",
+				`Execute the command with all root capabilities. This is similar to
+			running a command with "sudo" or executing "docker run" with the
+			"--privileged" flag. Containerization does not provide any security
+			guarantees when using this option. It should only be used when
+			absolutely necessary and only with trusted commands.`),
 
 		dagql.NodeFunc("terminal", s.terminal).
 			Doc(`Return an interactive terminal for this container using its configured default terminal command if not overridden by args (or sh as a fallback default).`).
-			ArgDoc("cmd", `If set, override the container's default terminal command and invoke these command arguments instead.`),
+			ArgDoc("cmd", `If set, override the container's default terminal command and invoke these command arguments instead.`).
+			ArgDoc("experimentalPrivilegedNesting",
+				`Provides dagger access to the executed command.`,
```

Comment:
```suggestion
				`Provides Dagger access to the executed command.`,
```

Reply:
It's written without `cap` on  `withExec` https://github.com/dagger/dagger/blob/583944d30e3b756bf880d78987402ca9796359f6/core/schema/container.go#L297-L298

I did that to keep the consistency, should I update it too in `withExec` then?

Reply:
Gotcha makes sense - thanks for updating that too!